packages/extension/src/companion/companionRequest.tsx

```diff
@@ -3,14 +3,18 @@ import { browser } from 'webextension-polyfill-ts';
 
 const proxyRequest = {
   apply(_, __, args) {
+    const initialCol = 'initial';
```

Comment:
I think we should take this outside the function and export it, so the reference will always be the same, even if we decide to change keys in the future, we will be safe. What do you think?

Reply:
Oh, yes. I totally forgot about that. Probably in one of our common files? Not exactly sure where the specific one would fit though.

Reply:
Oh. Really sorry. I meant to be placed in the shared package as well, so the key in the actual request would be able to utilize it as well. Sorry for the inconvenience 🙇

Reply:
Oh wait let me refactor I totally got your idea wrong 🤯

Reply:
Thank you! Sorry for lacking clarity! 🚢